test/infrastructure/docker/controllers/dockermachine_controller.go

```diff
@@ -223,16 +223,6 @@ func (r *DockerMachineReconciler) reconcileNormal(ctx context.Context, machine *
 }
 
 func (r *DockerMachineReconciler) reconcileDelete(ctx context.Context, machine *clusterv1.Machine, dockerMachine *infrav1.DockerMachine, externalMachine *docker.Machine, externalLoadBalancer *docker.LoadBalancer) (ctrl.Result, error) {
-	// if the deleted machine is a control-plane node, exec kubeadm reset so the etcd member hosted
```

Comment:
nit: what about adding a comment similar to what discussed in https://github.com/kubernetes-sigs/cluster-api/issues/2618#issuecomment-597147848